docs/ocis/deployment/systemd.md

```diff
@@ -36,19 +37,23 @@ Restart=always
 WantedBy=multi-user.target
 ```
 
-For reasons of simplicity we are using the root user and group to run oCIS which is not recommended. Please use a non-root user in production environments and modify the oCIS service definition accordingly.
-
+{{< hint danger >}}
+For reasons of simplicity we are using the root user and group to run oCIS, which is not recommended. Please use only privileged users in production environments and modify the oCIS service definition accordingly.
```

Comment:
We should definitely adapt our sample service file instead of this comment in the future.